sim/core/stats/stats.go

```diff
@@ -62,6 +62,7 @@ const (
 	HealingPower
 	SpellDamage
 	FeralAttackPower
+	Timeworn
```

Comment:
Don't add a stat for this. See the note below that explicitly says not to add stats :D 

If at all make a pseudo stat for it, or just a stand-alone function that counts valid equipped items and use that inside the aura handlers for the rings.

Reply:
Converted to a pseudostat